### PR TITLE
v_3_2_5: switch to hyperkube v1.10.1

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: v_3_2_5/master_template.go [hunk 1]
```diff
@@ -1827,13 +1827,13 @@ write_files:
...
-        - --insecure_bind_address=0.0.0.0
+        - --allow-privileged=true
+        - --insecure-bind-address=0.0.0.0
         - --anonymous-auth=false
         - --insecure-port=0
-        - --kubelet_https=true
+        - --kubelet-https=true
         - --kubelet-preferred-address-types=InternalIP
-        - --secure_port={{.Cluster.Kubernetes.API.SecurePort}}
+        - --secure-port={{.Cluster.Kubernetes.API.SecurePort}}
```
Comment (kopiczko) [inline v_3_2_5/master_template.go]: Those are deprecated we should replace them with some sort of config.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ version directory, and then changes are introduced.
 ## [v3.2.5] WIP
 
 ### Changed
+- Update hyperkube 1.10.1.
```
Comment (MarcelMue) [inline CHANGELOG.md]: Should this be `Updated` for consistency? Everything else is `Changed`, `Disabled`

Comment (kopiczko) [reply CHANGELOG.md]: Actually past tense is inconsistent.

Comment (puja108) [reply CHANGELOG.md]: Do we say hyperkube or do we just default to kubernetes, cause hyperkube might be confusing (it's an implementation detail) or we mention both?

File: v_3_2_5/master_template.go (hunk 1 above)
Comment (puja108) [reply v_3_2_5/master_template.go]: right kubelet is now dynamically configurable (you can even hotswap configuration IIRC) but it needs to be a config file I think

**Comment by kopiczko** [comment]
Tested on everything in the world.

File: CHANGELOG.md (hunk 2 above)
Comment (kopiczko) [reply CHANGELOG.md]: This is k8scloudconfig. Not exposed to the customer. I think implementation details are ok here

**Pull request merged**